RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCalendar/mobileAppointmentBookingCalendar.js

```diff
@@ -183,20 +183,22 @@ export default class MobileAppointmentBookingRebookingCalendar extends Lightning
    */
 
   showWeekView(firstdayOfWeek) {
+    var firstArr = [];
+    var tempRow = [];
     console.log("First day of the week is : " + firstdayOfWeek);
     this.boolShowPopover = false;
     this.noOfMonths = [];
-    var firstArr = [];
-    var tempRow = [];
+    firstArr = [];
```

Comment:
Why are these reassigned to empty values? They're assigned to empty values above, and not modified between there and here.

Reply:
@khawkins thanks,  I actually didn't refactor the entire inherited code only fixed lint errors. With time we will look into those but I'm currently working according to the team's priorities.